pants

```diff
@@ -72,42 +72,46 @@ EOF
 }
 
 # The high-level flow:
-# 1.) Resolve the Python interpreter, first reading from the env var $PYTHON,
+# 1.) Resolve the Pants version from pants.ini or from the latest stable
+#     release to PyPi.
+# 2.) Resolve the Python interpreter, first reading from the env var $PYTHON,
 #     then reading from pants.ini, then determining the default based off of
-#     which Python versions the current `pants_version` supports.
-# 2.) Grab pants version from pants.ini or default to latest.
+#     which Python versions the Pants version supports.
 # 3.) Check for a venv via a naming/path convention and execute if found.
```

Comment:
Could we maybe add a really quick description of why we need to resolve the pants version first now (I guess not just because we're using it resolve the python interpreter now, but also that it's mixed into the venv dir location)?

Reply:
Good idea! Fixed, along with updating some other parts of the docs so that we better explain the flow and fix things like link to the site.